app/services/grading_scale_service.rb

```diff
@@ -14,7 +14,14 @@ def count_for(grading_scale)
     if grading_scale.not_graded
       @term_registrations.ungraded.length
     else
-      range = (grading_scale.min_points..grading_scale.max_points)
+      if grading_scale.grade == "1"
```

Comment:
I'm not sure about checking for the grades this way. It might lead to problems later, if we were to change the grading scheme to "A-F" for example.

Instead we should check, if the given grading scale is equal to the first (or last) entry of the `@grading_scales` collection.